### DISCUSSION TITLE
Is there a guide?

### DISCUSSION THREAD

**Discussion by Lirloo** [rodit/SnapMod#231] [Q&A/Support]
Been trying to figure out how to use but I dont see this "menu" that everyone is talking about. I never own IOs. Can someone help? I have snapmods and the right supported version of snapchap

**Comment by Neeham** [comment]
IOs? Buddy, this works on Android only 😝

**Comment by Lirloo** [comment]
Yaa my dumbasss read something wrong. I manage to get stuff to work. My menu only have "save" even though I didn't hide the other. Is that normal atm?
